src/implementations/twig/components/text-input/text-input.test.js

```diff
@@ -1,10 +1,13 @@
 import { merge, renderTwigFileAsNode } from '@ecl/test-utils';
-import dataDefault from '@ecl/specs-component-text-input/demo/data';
+import specDefaultEc from '@ecl/specs-component-text-input/demo/data--ec';
+import specDefaultEu from '@ecl/specs-component-text-input/demo/data--eu';
 
-const dataInvalid = { ...dataDefault, invalid: true };
-const dataDisabled = { ...dataDefault, disabled: true };
+const specInvalidEc = { ...specDefaultEc, invalid: true };
+const specDisabledEc = { ...specDefaultEc, disabled: true };
+const specInvalidEu = { ...specDefaultEu, invalid: true };
+const specDisabledEu = { ...specDefaultEu, disabled: true };
 
-describe('Text field', () => {
```

Comment:
The structure of the test is a bit weird, i guess it was already, but we now added EU and they doubled, we test again the `extra whatever` for the `disabled` and `invalid` use case, which looks redundant, no reason why these would fail if the default doesn't. Wouldn't it be better to have also the `disabled` and the `invalid` as the `required` currently is, basically? Instead of "describing" them as `Default` we could describe them as `Text input EC` and `Text input EU` directly, also...